update MNG pod drain timeout (#1128)
awsdocs/amazon-eks-user-guide: latest/ug/nodes/delete-managed-node-group.adoc

This topic describes how you can delete an Amazon EKS managed node group. When you delete a managed node group, Amazon EKS first sets the minimum, maximum, and desired size of your Auto Scaling group to zero. This then causes your node group to scale down.

> Before each instance is terminated, Amazon EKS sends a signal to drain the Pods from that node. If the Pods haven't drained after a 5 minutes, Amazon EKS lets Auto Scaling continue the force termination of the instance. After every instance is terminated, the Auto Scaling group is deleted.

Reviewer: ```suggestion
Before each instance is terminated, Amazon EKS sends a signal to drain that node. If the node hasn't been drained after 5 minutes, Amazon EKS lets Auto Scaling continue the forced termination of the instance. After all instances have been terminated, the Auto Scaling group is deleted.
```
Traditionally we use `drain` for Nodes and `evict` for Pods, and I think `drain that node` reads better than `evict Pods from that node`.

Author: Updated drain for nodes and evict for Pods. also included preStop lifecycle hooks.

> Before each instance is terminated, Amazon EKS sends a signal to drain that node. During the drain process, Kubernetes executes any configured preStop lifecycle hooks, sends SIGTERM signals to containers, and respects the pod's terminationGracePeriodSeconds. If the node hasn't been drained after 5 minutes, Amazon EKS lets Auto Scaling continue the forced termination of the instance. After all instances have been terminated, the Auto Scaling group is deleted.

Reviewer 2: ```suggestion
Before each instance is terminated, Amazon EKS sends a signal to drain that node. During the drain process, Kubernetes does the following for each pod on the node: waits for the `terminationGracePeriodSeconds`, runs any configured `preStop` lifecycle hooks, then sends `SIGTERM` signals to the containers. If the node hasn't been drained after 5 minutes, Amazon EKS lets Auto Scaling continue the forced termination of the instance. After all instances have been terminated, the Auto Scaling group is deleted.
```

Author: done

Reviewer 3: The order of events seems incorrect to me. The `terminationGracePeriodSeconds` starts after `SIGTERM` signal is sent. See this for ref: https://cloud.google.com/blog/products/containers-kubernetes/kubernetes-best-practices-terminating-with-grace

so this should read as:
```
For each pod on the node, Kubernetes does the following: (1) executes any configured preStop lifecycle hooks, (2) sends SIGTERM signals to the containers, (3) waits for the terminationGracePeriodSeconds for the containers to shut down gracefully.
```

Author: ack, done
